test/spec/algoliasearch.helper/client.js

```diff
@@ -1,113 +1,102 @@
 'use strict';
 
-var test = require('tape');
-var sinon = require('sinon');
 var algoliaSearchHelper = require('../../../index.js');
 var version = require('../../../src/version');
 var algoliasearch = require('algoliasearch');
 
 function makeFakeClient() {
   var client = algoliasearch('what', 'wait', {});
-  client.search = sinon.stub().returns(new Promise(function() {}));
+
+  client.search = jest.fn(function() {
+    return new Promise(function() {});
+  });
+
   return client;
 }
 
-test("client without addAlgoliaAgent() doesn't throw on instanciation", function(t) {
+test("client without addAlgoliaAgent() doesn't throw on instanciation", function() {
   var client = {};
 
-  t.doesNotThrow(function() {
+  expect(function() {
     algoliaSearchHelper(client);
-  });
-
-  t.end();
+  }).not.toThrow();
 });
 
-test('addAlgoliaAgent gets called if exists', function(t) {
+test('addAlgoliaAgent gets called if exists', function() {
   var client = {
-    addAlgoliaAgent: sinon.stub()
+    addAlgoliaAgent: jest.fn()
   };
 
-  t.notOk(client.addAlgoliaAgent.called);
+  expect(client.addAlgoliaAgent).not.toHaveBeenCalled();
```

Comment:
We can use `toHaveBeenCalledTimes(0)` for better error messages and easier refactors.

Reply:
Nope we can't rely on the number of call for this test. It is actually called multiple times but it's an implementation detail that we don't care about (the client deduplicate them). We could use `toHaveBeenCalledTimes` for the first one but not the second. To be consistent for the two assertions I've used `toHaveBeenCalled` on both.